### PR TITLE
Find a wallet table: fade icons in and out when dropdown select is changed

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Adds functionality in for fading out and in the feature icons for a wallet in the WalletTable component.

### CODE REVIEW

**Comment by corwintines** [comment]
Open to changing the timing of this animation. Some user feedback from user testing suggested that the compare features didn't seem like they were doing anything because the icons weren't updating (they were, just didn't look like it since they didn't change). This adds a fade-in and out animation to give the appearance of things changing, even if it's just going from true for one feature to true for another.

File: src/components/FindWallet/WalletTable.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Libraries
-import React, { useState } from "react"
+import React, { useState, SVGProps } from "react"
 import { getImage, GatsbyImage } from "gatsby-plugin-image"
+import { keyframes } from "@emotion/react"
```
Comment (minimalsm) [inline src/components/FindWallet/WalletTable.tsx:4]: Should we decide on a convention of how we animate (maybe framer-motion?) on ethereum.org?

We now have at least three ways to animate:

- motion components
- CSS animations
- keyframes

File: src/components/FindWallet/WalletTable.tsx [hunk 2]
```diff
@@ -250,11 +251,34 @@ const FlexInfo = styled.div`
...
+
 const FlexInfoCenter = styled(FlexInfo)`
   justify-content: center;
   cursor: pointer;
   height: 100%;
   display: flex;
+  animation: ${fadeIn} 0.5s;
+
+  &.fade {
+    animation: ${fadeOut} 0.5s;
```
Comment (minimalsm) [inline src/components/FindWallet/WalletTable.tsx:280]: It's a little too much. Maybe try half of this?

```suggestion
  animation: ${fadeIn} 0.25s;

  &.fade {
    animation: ${fadeOut} 0.25s;
```

**Review by minimalsm — COMMENTED** [review]
Nice :-)

Agree on changing the timing... it feels a little too dramatic as is 😅.

Also left a comment about us picking an animation convention and but not necessarily a blocker to this PR.

File: src/components/FindWallet/WalletTable.tsx (hunk 2 above)
Comment (corwintines) [reply src/components/FindWallet/WalletTable.tsx:280]: I was thinking of shortening the time to something like this, will bring this in :)

File: src/components/FindWallet/WalletTable.tsx (hunk 1 above)
Comment (corwintines) [reply src/components/FindWallet/WalletTable.tsx:4]: Agree. I know that we use framer for a lot of other animations so will refactor to that.

**Review by minimalsm — APPROVED** [review]
LGTM. 

@nloureiro or @konopkja to review?

**Comment by konopkja** [comment]
looks good to me

**Comment by nloureiro** [comment]
> LGTM.
> 
> @nloureiro or @konopkja to review?

Just to be sure, are we working on this fading?

https://user-images.githubusercontent.com/1120748/193794839-735f879e-98db-4d5d-b06f-3493d51c591c.mov

If yes, it looks good!

**Comment by minimalsm** [comment]
@nloureiro yes :-)! thanks guys, merged!

**Pull request merged**